### ISSUE TITLE
command line cole

### ISSUE THREAD

**Issue by ColeWill** [den-materials/html-forms#1]
link 1
link 2

asked will

my command line is broke

**Comment by ColeWill** [comment]
When I tried to push my files to git hub they wouldn’t go, because I made some accidental nested git repos all over my computer…. BUT stefan found that if you simply re-clone the file and add a “.” to the end of the link "https://github.com/CoolCode2/css-html-forms-lab.git ."

Then your clone will work and it won't reference the other repos that aren't working.

**Comment by stefems** [comment]
Those links aren't links... Highlight the text, click the link button in the editing panel above the text entry area, and paste the link into the parentheses!

What makes you think that your command line is broken? Is your whole command line broken? Or is it just a command you're typing that is broken? This distinction is important! Understanding the solution to a problem _requires a detailed look into the problem!_

We used this [stack question](http://stackoverflow.com/questions/7705499/git-status-shows-untracked-file-on-a-higher-level) to fix Cole's issue:

We used this command to clone specifically into the current folder:
`git clone https://github.com/CoolCode2/css-html-forms-lab.git .`

Don't do `git init` in folders you don't want to be repos, and delete the git config files to get rid of the git configuration. Don't nest git repos inside of git repos. That's gonna cause some complexity we don't want.
